modules/local/nacho/normalize/tests/main.nf.test

```diff
@@ -0,0 +1,35 @@
+nextflow_process {
+
+    name "Test Process NACHO_NORMALIZE"
+    script "../main.nf"
+    process "NACHO_NORMALIZE"
+
+    test("Should run without failures") {
+
+        when {
+            params {
+                pipelines_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/'
```

Comment:
This is already defined in `nextflow.config`. See:
https://github.com/nf-core/nanostring/blob/e765843c16d2bfe90c72e31a7a3719466d1bdfb0/nextflow.config#L51

Reply:
@atrigila Is it ok to rely on this nextflow.config file? What if you're trying to test the module alone or in another pipeline where this parameter is not defined?

Reply:
In this case, this is a local module for this pipeline. The definition of the `pipelines_testdata_base_path` in `nextflow.config` is done to avoid repetition of datasets in test configs. (see its introduction in: https://github.com/nf-core/website/blob/f207b76d2896ae55ceaee4f9f834cd5db43a6eb5/sites/main-site/src/content/blog/2024/tools-2_14.mdx?plain=1#L128)

If you were to do this for nf-core/modules, you would explicitly add which is the modules/pipelines test path or you would add the full path.